apps/website/vercel.json

```diff
@@ -2,5 +2,16 @@
   "$schema": "https://openapi.vercel.sh/vercel.json",
   "ignoreCommand": "git diff --quiet HEAD^ HEAD ../../{patches,package.json,turbo.json} ../../packages/{colors,components,icons,status-js} ./",
   "installCommand": "yarn install --cwd ../../ --frozen-lockfile",
-  "buildCommand": "turbo run build --cwd ../../ --filter=website..."
+  "buildCommand": "turbo run build --cwd ../../ --filter=website...",
+  "headers": [
+    {
+      "source": "/.well-known/apple-app-site-association",
+      "headers": [
+        {
+          "key": "Content-Type",
+          "value": "application/json"
+        }
+      ]
+    }
+  ]
```

Comment:
try https://vercel.com/guides/how-to-configure-the-cache-control-response-header-in-vercel-projects#using-the-next.config.js-file again after fixing typo

Reply:
@prichodko per double definition, ditto. tldr: typo, will try again without one